File: 0179-largest-number/0179-largest-number.ts
function largestNumber(nums: number[]): string {
  const arrString = nums.map(String).sort(lambdaSort);
  return arrString[0] === '0' ? '0' : arrString.join('');
};

function lambdaSort(a, b) {
  return (b + a) - (a + b);
};

File: 0179-largest-number/0179-largest-number.py
class Solution:
    def largestNumber(self, nums: List[int]) -> str:
      strings_list = list(map(str, nums))
      
      strings_list.sort(key=lambda a: a * 10, reverse=True)
      
      if strings_list[0] == '0':
        return '0'
      
      return "".join(strings_list)
        